Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/PromptCodeReceiver.cs

```diff
@@ -41,7 +40,13 @@ public string RedirectUri
             var authorizationUrl = url.Build().ToString();
 
             Logger.Debug("Open a browser with \"{0}\" URL", authorizationUrl);
```

Comment:
OK, done.
I was kind of thinking that the wording was ambiguous enough to leave it in both ;)